ui/app/src/components/DeleteProjectDialog/DeleteProjectDialog.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { DispatchWithoutAction } from 'react';
+import { Dialog, DialogTitle, DialogContent, DialogActions, Button, IconButton } from '@mui/material';
+import CloseIcon from 'mdi-material-ui/Close';
+
+export interface DeleteProjectDialogProps {
+  name: string | undefined;
```

Comment:
Another place I'd be in favor of `projectName` to be more explicit. But, this is very straightforward too - I don't think there would be a lot of confusion, so I think this is fine too

Reply:
yeah I think I prefer name as the context is obvious in that file.